migrations/1570084134123_add-pending-snapshots.js

```diff
@@ -0,0 +1,16 @@
+exports.shorthands = undefined
+
+const SQL = `
+CREATE TABLE pending_snapshots  (
```

Comment:
Let's rename this table into `transient_snapshots`, because we will be using it for both pending and failed transactions.